addOns/postman/src/test/java/org/zaproxy/addon/postman/ItemDeserializerUnitTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2023 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.postman;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.zaproxy.addon.postman.Models.ItemGroup;
+import org.zaproxy.addon.postman.Models.PostmanCollection;
+import org.zaproxy.addon.postman.Models.SingleItem;
+import org.zaproxy.zap.testutils.TestUtils;
+
+public class ItemDeserializerUnitTest extends TestUtils {
+
+    @BeforeEach
+    void setup() throws Exception {
+        setUpZap();
+        startServer();
+    }
+
+    @AfterEach
+    void teardown() throws Exception {
+        stopServer();
+    }
```

Comment:
I'd not expect these to be needed.

Reply:
It's giving `java.lang.NullPointerException: Cannot invoke "org.parosproxy.paros.control.Control.getExtensionLoader()" because the return value of "org.parosproxy.paros.control.Control.getSingleton()" is null` without these.

Reply:
Only `setUpZap()` is necessary for that.